tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "outDir": "./dist/out-tsc",
     "forceConsistentCasingInFileNames": true,
     "strict": true,
+    "strictNullChecks":false,
```

Comment:
Why mate? This is not the solution. This is like put the trash under the carpet of your living. Strict null checks makes your code a lot more robust. 
Doing this you are taking an architectural decision, basically, you are setting the quality bar of your project a step-down.

As this is a whiteboard-clean project from the beginning, you must keep the quality where it was. 
Set this setting to true and fix the issues, please.

Reply:
I am really struggling to fix this issue, I would appreciate a help from you. Thansk!

Reply:
OK, what's the problem?  You must set this to true, then build and fix each strict comparison (null check) you have on the code. the compiler should tell you the file and line where the error occurs?